app/main/forms/auth_forms.py

```diff
@@ -44,14 +44,23 @@ class ChangePasswordForm(FlaskForm):
         ]
     )
     confirm_password = PasswordField(
-        'Confirm password', id="input_confirm_password",
+        'Confirm new password', id="input_confirm_password",
         validators=[
             DataRequired(message="Please confirm your new password"),
             EqualTo('password', message="The passwords you entered do not match")
         ]
     )
 
 
+class ChangeOldPasswordForm(ChangePasswordForm):
+    old_password = PasswordField(
```

Comment:
Per @Wynndow's comments, I think I agree that this form isn't named very clearly - it could either do with a different name or with a very brief docblock explaining what it's for.

Why doesn't the ChangePasswordForm always ask for the old password? Maybe it needs renaming, if it's actually only used when the account is first created? Some confusion here anywhere that could be to do with naming things better.

Reply:
Yeah, it's a bit confusing. We were reluctant to change the password reset flow too much but `ChangePasswordForm` is no longer the right name for that form! I'll take a look at that now.